src/main/java/com/solvd/laba/carina/homework/pages/network_configuration_tool/Login.java

```diff
@@ -0,0 +1,52 @@
+package com.solvd.laba.carina.homework.pages.network_configuration_tool;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.decorator.PageOpeningStrategy;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class Login extends AbstractPage {
+    @FindBy(css="#txt_Username")
+    private ExtendedWebElement usernameInput;
+
+    @FindBy(css="#txt_Password")
+    private ExtendedWebElement passwordInput;
+
+    @FindBy(css="#loginbutton")
+    private ExtendedWebElement loginButton;
+
+    @FindBy(css="#DivErrPromt")
+    private ExtendedWebElement loginErrorMessage;
+
+    public Login(WebDriver driver) {
+        super(driver);
+        setPageOpeningStrategy(PageOpeningStrategy.BY_URL);
+        setPageAbsoluteURL("http://192.168.1.254/");
+    }
+
+    public void enterUsername(String username) {
+        usernameInput.type(username);
```

Comment:
Small and atomic methods are an excellent approach, but for convenience, you can also create one method that contains a call to all these methods. You can call it login and pass a User object to it